core/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -426,40 +453,24 @@ private[spark] class SparkArrayIndexOutOfBoundsException(
  */
 private[spark] class SparkSQLException(
```

Comment:
This is used ONLY once, and tested by QueryExecutionErrorsSuite "UNRECOGNIZED_SQL_TYPE: unrecognized SQL type -100"